skypy/galaxies/vdf_sampler.py

```diff
@@ -0,0 +1,92 @@
+"""Sample from velocity distribution function
+"""
+
+import numpy as np
+import scipy.special as special
+from scipy.interpolate import interp1d
+
+
+def sample_vdf(x_min, x_max, resolution=100, size=1):
```

Comment:
I suggest not using `sample` in the name of the function. We have similar functions for luminosities and redshifts that sample from a Schechter function. See main comment.

Reply:
Done.